testsuite: please skip dead-debug-sections if dwarfdump is not installed

I miss the tool and so I face the following error:

```
Testing dead-debug-sections ... ./elf/dead-debug-sections.sh: line 40: dwarfdump: command not found
make[1]: *** [Makefile.linux:6: elf/dead-debug-sections.sh] Error 127
make[1]: *** Waiting for unfinished jobs....
```

--- a/test/elf/dead-debug-sections.sh
+++ b/test/elf/dead-debug-sections.sh
@@ -14,6 +14,8 @@ mold="$(pwd)/mold"
 t=out/test/elf/$testname
 mkdir -p $t
 
+which dwarfdump >& /dev/null || { echo skipped; exit; }
+
 cat <<EOF | $CXX -c -o $t/a.o -g -xc++ -
 #include <iostream>
 struct Foo {

